.github/scripts/config.ps1

```diff
@@ -7,8 +7,8 @@ if ( -not $env:LIBJPEG_TURBO_VERSION) { $env:LIBJPEG_TURBO_VERSION="libjpeg-turb
 if ( -not $env:ARCH ) { $env:ARCH = "aarch64" }
 if ( -not $env:PLATFORM ) { $env:PLATFORM = "w64-mingw32" }
 if ( -not $env:CRT ) { $env:CRT = "msvcrt" }
-if ( -not $env:TARGET ) { $env:TARGET = "$ARCH-$PLATFORM" }
-if ( -not $env:TOOLCHAIN_NAME ) { $env:TOOLCHAIN_NAME = "$ARCH-$PLATFORM-$CRT" }
+if ( -not $env:TARGET ) { $env:TARGET = "$env:ARCH-$env:PLATFORM" }
+if ( -not $env:TOOLCHAIN_NAME ) { $env:TOOLCHAIN_NAME = "$env:ARCH-$env:PLATFORM-$env:CRT" }
```

Comment:
what about using TARGET?

Reply:
Can you describe the use case? I understand `$env:ARCH`, `$env:PLATFORM`, and `$env:CRT` as primary way of defining the target and the `$env:TARGET` as rather read-only shortcut or at most "optional override".

Reply:
I was suggesting to use TARGET instead of $env:ARCH-$env:PLATFORM in TOOLCHAIN_NAME definition.

Reply:
I understand that. I am just suggesting that `$env:ARCH-$env:PLATFORM-$env:CRT` triplet should have precedence over `$env:TARGET` that can be overriden unless there is any specific use-case I am missing?

Reply:
it will be strange to have different naming for target and toolchain name. Up to you, it was just a suggestion.

Reply:
I am thinking of the use cases where the most of the toolchain components are being build using the `$env:ARCH-$env:PLATFORM-$env:CRT` triplet but, e.g., one particular component for one particular target needs to have `$env:TARAGET` variable overriden, e.g. `pc-linux-gnu` vs `linux-gnu`.